assets/js/components/AboutProfiles.js

```diff
@@ -11,11 +11,8 @@ module.exports = class AboutProfiles {
     this.window = _window;
     this.doc = doc;
 
-    this.$elm.classList.add('grid-listing');
+    this.$elm.classList.add('grid-listing__about');
```

Comment:
are there styles that we can borrow from `grid-listing` too and this can be a variant?

Reply:
I think I have used all I can apart from the breakpoint. I will have another look this morning.

Reply:
I have created three mixins so these can be used between both grid-listing and grid-listing__about.